.travis.yml

```diff
@@ -491,3 +549,11 @@ jobs:
       - kubectl get pods --all-namespaces
       - kubectl get services --all-namespaces
       - kubectl get ingress --all-namespaces
+
+  - <<: *minikubeStandaloneTest
+    env:
+      - MINIKUBE_VERSION=v1.3.1
+
+  - <<: *minikubeStandaloneTest
+    env:
+      - MINIKUBE_VERSION=v1.4.0
```

Comment:
Am I right that:
- Minikube v1.3.1 runs K8s 1.15
- Minikube v.1.4.0 runs K8s 1.16
as shown here: https://github.com/kubernetes/minikube/blob/v1.4.0/CHANGELOG.md 

If yes, I will take care of updating the Minikube column of the Keptn/K8s support matrix: https://github.com/keptn/keptn.github.io/pull/539

Reply:
Versions seem to be correct - though this only affects the control-plane installation (not delivery).

Reply:
thanks for clarification!